### PR TITLE
implementing Stack using Queue

### PR DESCRIPTION
**Pull request by PankajKumarAgrawal1729**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

### CODE REVIEW

**Comment by PankajKumarAgrawal1729** [comment]
Hi Guys,
Please review my PR.
@yanglbme @debasishbsws @vil02 @BamaCharanChhandogi

File: src/main/java/com/thealgorithms/stacks/StackUsingQueue.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        queue2.add(val);
+    }
+
+    // Function to add an element to the stack, which is implemented by queue1
+    public void push(Object x) {
+        queue1.add(x);
+    }
+
+    // Function to remove the top element from the stack
+    public Object pop() {
```
Comment (susanhsrestha) [inline src/main/java/com/thealgorithms/stacks/StackUsingQueue.java:39]: Exception handling for the pop function if no elements is present is required.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: src/main/java/com/thealgorithms/stacks/StackUsingQueue.java (hunk 1 above)
Comment (PankajKumarAgrawal1729) [reply src/main/java/com/thealgorithms/stacks/StackUsingQueue.java:51]: Hi @susanhsrestha,
i added the Exception handling.
Please check.

Comment (susanhsrestha) [reply src/main/java/com/thealgorithms/stacks/StackUsingQueue.java:51]: LGTM!

**Review by susanhsrestha — APPROVED** [review]

**Comment by PankajKumarAgrawal1729** [comment]
Hi @yanglbme @debasishbsws @vil02 @BamaCharanChhandogi ,
Please reviewed my PR and merged it.

**Review by siriak — CHANGES_REQUESTED** [review]
The code looks good, could you please remove `main` and add some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

**Pull request closed without merging**